src/main/java/com/github/manolo8/darkbot/utils/login/LoginUtils.java

```diff
@@ -158,10 +166,16 @@ private static void usernameLogin(LoginData loginData, String domain) throws IOE
 
         CookieHandler.setDefault(null);
 
-        HttpCookie cookie = cookieManager.getCookieStore().getCookies().stream()
+        HttpCookie cookie;
+
+        if (captcha && domain.equals("lp")) {
+            Application.launch(LoginBrowser.class, loginData.getUsername(), loginData.getPassword());
+        }
+
+        cookie = cookieManager.getCookieStore().getCookies().stream()
                 .filter(c -> c.getName().equalsIgnoreCase("dosid"))
                 .filter(c -> c.getDomain().matches(".*\\d+.*"))
-                .findFirst().orElseThrow(() -> new WrongCredentialsException("Wrong credentials or unsolved reCaptcha"));
+                .findFirst().orElse(null);
```

Comment:
Don't touch this stuff, leave it be as it was, this is for logging in, which you shouldn't need to mess with. You should only need to mess with the captcha, which is an "alternative" to logging in, you don't need to mess with this code to implement your thing